packages/mobile/src/escrow/EscrowedPaymentListItem.tsx

```diff
@@ -125,5 +148,7 @@ const styles = StyleSheet.create({
 })
 
 export default componentWithAnalytics(
-  withTranslation(Namespaces.inviteFlow11)(EscrowedPaymentListItem)
+  connect<StateProps, {}, {}, RootState>(mapStateToProps)(
```

Comment:
This isn't a hard rule, but for things that might appear in a list it is often nice to only connect to redux at the "list" level and then pass down relevant state/props. So in this case we probably won't have that many, but just an fyi

Reply:
I think I follow conceptually but not sure how to execute. Is there an example in the code you could point me to?

Reply:
+1 for avoiding connecting list items (components that get rendered as items in a flat list).
The solution would be to connect this into the parent component and pass in the data as a prop

Reply:
Done